### PR TITLE
Add run from distgit to docs

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- Mention running packit from distgit.

### CODE REVIEW

**Comment by lachmanfrantisek** [comment]
I have some problems with the regeneration so I am posting only the content change for review...

**Comment by lachmanfrantisek** [comment]
Please, wait with the merge to https://github.com/packit-service/packit/pull/534

File: content/docs/configuration.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ Both Packit-as-a-Service and packit tool use this configuration file.
 ---------------------------|-----------------|----------------------------------------------------------------------
  `specfile_path`           | string          | relative path to a spec file within the upstream repository (mandatory)
  `upstream_project_name`   | string          | name of the upstream repository (e.g. in PyPI); this is used in `%prep` section to generate an archive
+ `upstream_project_url`    | string          | URL of the upstream prject (e.g. `https://github.com/packit-service/packit`)
```
Comment (jpopelka) [inline content/docs/configuration.md]:
```suggestion
 `upstream_project_url`    | string          | URL of the upstream project (e.g. `https://github.com/packit-service/packit`)
```

File: content/docs/guide.md [hunk 2]
```diff
@@ -74,6 +74,12 @@ You can also read more about the SRPM process in [the document]({{< ref
 Is your `packit srpm` finally passing? If not, feel free to reach out to us. If
 yes, let's proceed to the next level.
 
+### Running from the dist-git repository
+
+When you are not allowed or do not want to run packit command from the upstream git repository,
+you can run commands from the cloned dist-git repository in a same way as you do from upstream.
```
Comment (TomasTomecek) [inline content/docs/guide.md]:
```suggestion
you can run commands from the cloned dist-git repository in the same way as you do from upstream.
```

**Comment by lachmanfrantisek** [comment]
@jpopelka Thanks for merging.

**Pull request merged**